core/src/components/text-area/text-area.stories.tsx

```diff
@@ -1,15 +1,22 @@
-import { storiesOf } from "@storybook/react";
+import { _Story } from "../../_story";
 import { DivPx } from "../div/div";
 import { TextArea } from "./text-area";
 
-storiesOf("TextArea", module).add("Main", () => (
-	<div style={{ width: 300 }}>
-		<TextArea />
-        <DivPx size={8} />
-		<TextArea placeholder="Placeholder" />
-        <DivPx size={8} />
-		<TextArea readOnly value="Read-only" />
-        <DivPx size={8} />
-		<TextArea disabled />
-	</div>
-));
+export default {
+	title: "Components/TextArea",
+	component: TextArea,
+};
+
+export const Primary = () => {
+	return (
+		<div style={{ width: 300 }}>
+			<TextArea />
+			<DivPx size={8} />
+			<TextArea placeholder="Placeholder" />
+			<DivPx size={8} />
+			<TextArea readOnly value="Read-only" />
+			<DivPx size={8} />
+			<TextArea disabled />
+		</div>
+	);
+};
```

Comment:
Usually the Primary should showcase only 1 component, and take advantage of Storybook's [Controls](https://storybook.js.org/docs/react/essentials/controls) for simple props (such as disabled, style and size)